srcpkgs/nagstamon/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'nagstamon'
+pkgname=nagstamon
+version=3.0.2
+revision=1
+wrksrc=Nagstamon
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-keyring"
+depends="python3-BeautifulSoup4 python3-dbus python3-keyring python3-lxml python3-psutil python3-PyQt5-multimedia python3-PyQt5-svg python3-requests"
+short_desc="Nagios status monitor for the desktop"
+maintainer="Laszlo Dvornik <laulicus@zoho.com>"
+license="GPL-2"
```

Comment:
You should use the [SPDX Identifier](https://spdx.org/licenses/) in this case `GPL-2.0-or-later`

Reply:
In the source tarball, I find this is only GPL2. GPL-2.0-only would be OK?

Reply:
Take a look at the headers of the source files. e.g. https://github.com/HenriWahl/Nagstamon/blob/master/Nagstamon/__init__.py

Reply:
Thank you. Fixed.